release-team/role-handbooks/bug-triage/README.md

```diff
@@ -55,233 +33,208 @@ Bug Triage has a lower time requirement than most roles at the beginning of the
-  * `is:open is:pr created:>2018-06-14` [open PRs, no exclusions, created since you last looked](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen++created%3A%3E2018-06-14)
-  * `is:open is:pr modified:>2018-06-14` [open PRs, no exclusions, modified since you last looked](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen++modified%3A%3E2018-06-14)
-* Reports against the beta(s):
-  * `is:issue "1.14.0-beta"' [issues mentioning beta version](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=%221.14.0-beta%22)
-  * `is:pr "1.14.0-beta"' [PRs mentioning beta version](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=%221.14.0-beta%22)
+If the issue/PR has the `priority/critical-urgent` label, it should be coordinated with the owning SIG and the Release Team Lead should it be allowed to stay or should the priority be decreased and issue/PR moved to the next milestone.
 
-### Reports [Freeze]
+#### Reports
 
-A few days after Code Freeze kicks in, daily 'burndown' meetings start, which means that either you or a shadow has to give updates every day. See [Issue Categorization](#issue-categorization).
+At this point, the report is contained of the following items:
+- Total number of non-test/flake related issues, number of `priority/critical-urgent` and `priority/important-soon` issues
+  - If there are `priority/critical-urgent` issues, those should be listed
+- Total number of PRs and number of `priority/critical-urgent` and `priority/important-soon` PRs
+  - If there are `priority/critical-urgent` PRs, those should be listed
+- Total number of issues, including flake/test related ones
 
-There will also be several enhancements which will have requested exceptions from the normal release timeline/requirements.  You'll want to track the exceptions specifically and report on them.  It is useful to be tracking in a spreadsheet the open issues/PRs which have caught your attention, because GitHub queries are only point in time.  It's easy for an issue you want to follow to stop showing up in a query and thus fall off your radar if you're only going by the queries.
+If there are issues/PRs that are going to block the release (i.e. `priority/critical-urgent` issues) and those are not close to being merged (e.g. in the merge pool or in-review process) that status is usually red. Otherwise, the status is yellow (if such issues/PRs are close to being merged) or green (if we don't have such issues/PRs).
 
-## Code Thaw
+### Code Thaw
 
-Starting a few working days before the first Release Candidate, Code Freeze labeling restrictions are lifted.  At this point, you need to make sure that three things happen:
+Starting a week before the stable release, Code Freeze labeling restrictions are lifted. At this point, you need to make sure that three things happen:
```

Comment:
maybe add the specific tag for stable release?

Reply:
Can you please give me an example for this?

Reply:
Apologies - I thought we were talking about a pre-release here - since "stable" can mean different things, maybe jsut drop "stable" in favor of "a week before the release target date".

Reply:
This is now fixed.